src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -343,7 +346,7 @@ Integer insertDataset(
         "SELECT d.* from dataset d " +
             " INNER JOIN consentassociations a ON d.dataset_id = a.datasetid " +
             " INNER JOIN consents c ON a.consentId = c.consentId " +
-            " WHERE c.dac_id IS NULL ")
+            " WHERE d.dac_id IS NULL ")
     List<Dataset> findNonDACDatasets();
```

Comment:
I don't think we need the inner joins on `consents` or `consentassociations` any more.